.github/workflows/build.yml

```diff
@@ -116,11 +116,27 @@ jobs:
           repo sync -c --no-repo-verify --force-sync -v
           repo forall -c 'git lfs pull'
 
+      - name: Cache restore OpenHarmony prebuilts
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+             /__w/manifest/manifest/openharmony_prebuilts
+          key:
+            cache-openharmony-prebuilts-${{ hashFiles('**/*.xml') }} 
+          restore-keys:
+            cache-openharmony-prebuilts-
 
       - name: Download prebuilts
         run: |
           cd repo && ./build/prebuilts_download.sh
```

Comment:
I think that it shouldn't be downloading anything again when it is cached, the few minutes it takes should be because it's looking up the checksums to verify everything is in place. Also the script does the installation of the prebuilts into the source tree

Reply:
ok